plugins/Marketplace/stylesheets/plugin-details.less

```diff
@@ -102,5 +97,249 @@
     img {
         max-width: 100%;
     }
+}
+
+// plugin details modal
+#pluginDetailsModal {
+    min-height: 50vh;
+    max-height: 80vh;
+    width: 60%;
+    color: #000;
+
+    .btn-close {
+        z-index: 100;
+    }
+
+    .star-icon {
+        height: 24px;
+        width: 24px;
+        margin-right: 4px;
+        vertical-align: unset;
+    }
+
+    .modal-content {
+        display: flex;
+        flex-direction: column;
+        flex: 1;
+        padding: 0;
+        min-height: 50vh;
+        max-height: 80vh;
+    }
+
+    .modal-content__header {
+        padding: 24px 24px 0;
+        height: 160px;
```

Comment:
Who wouls have guessed, but once a plugin title breaks into a second line ("Multi Channel Conversion Attribution" is a good candidate for that) the elements below overlap one another.

Would that be reasonably prevented just removing that setting here? In combination with not removing the bottom padding (so just `padding: 24px` above) it looked okay, haven't checked in various browsers though.

Reply:
Kept the height and adjusted the font-size and spacing on narrower screens as the height is used to calculate the height of the content.